src/WebUI/src/pages/moderator/logs.vue

```diff
@@ -0,0 +1,241 @@
+          />
+        </OField>
+
+        <OField label="From">
+          <ODateTimePicker
+            v-model="from"
+            size="sm"
+            locale="en"
+            clearable
+            iconRight="calendar"
+            datepickerWrapperClass="w-44"
+          />
+        </OField>
+
+        <OField label="To">
+          <ODateTimePicker
+            v-model="to"
+            size="sm"
+            locale="en"
+            :max="new Date()"
+            iconRight="calendar"
+            datepickerWrapperClass="w-44"
+          />
+        </OField>
+
+        <div class="flex items-end gap-4">
+          <VDropdown :triggers="['click']">
+            <template #default="{ shown }">
+              <OInput
+                :modelValue="types.join(',')"
+                type="text"
+                size="sm"
+                placeholder="Type"
+                :icon="shown ? 'chevron-up' : 'chevron-down'"
+                :iconRight="types.length !== 0 ? 'close' : undefined"
+                iconRightClickable
+                readonly
+                @iconRightClick.stop="types = []"
+                class="w-44 cursor-pointer overflow-x-hidden text-ellipsis"
+              />
+            </template>
+            <template #popper>
+              <div class="max-h-60 min-w-60 max-w-xs overflow-y-auto">
+                <DropdownItem v-for="activityLogType in Object.keys(ActivityLogType)">
+                  <OCheckbox v-model="types" :native-value="activityLogType">
+                    {{ activityLogType }}
+                  </OCheckbox>
+                </DropdownItem>
+              </div>
+            </template>
+          </VDropdown>
+
+          <OButton size="sm" label="Search" variant="primary" @click="search" />
+        </div>
+      </OField>
+
+      <div>
+        <OTable
+          :data="activityLogs"
+          v-model:current-page="pageModel"
+          :perPage="perPage"
+          :paginated="activityLogs.length > perPage"
+          hoverable
+          bordered
+          detailed
+          detailKey="id"
+          :hasDetailedVisible="hasDetailedVisible"
+          showDetailIcon
+          :debounceSearch="300"
+          sortIcon="chevron-up"
+          sortIconSize="xs"
+          :loading="isLoading"
+          :defaultSort="['id', 'desc']"
+        >
+          <OTableColumn
+            #default="{ row: activityLog }: { row: ActivityLog }"
+            field="type"
+            label="Type"
+            sortable
+            width="200"
+          >
+            <Tag :label="activityLog.type" />
+          </OTableColumn>
+
+          <OTableColumn
+            #default="{ row: activityLog }: { row: ActivityLog }"
+            field="createdAt"
+            label="Created at"
+            sortable
+          >
+            {{ $d(activityLog.createdAt, 'long') }}
+          </OTableColumn>
+
+          <OTableColumn
+            #default="{ row: activityLog }: { row: ActivityLog }"
```

Comment:
OPT: add an interaction on a cell to "filter by this user", "filter by this type"